packages/ProgressIndicators/src/ProgressBar/README.md

```diff
@@ -0,0 +1,59 @@
+# ProgressBar
```

Comment:
Is this README supposed to be in both package root and component directories? Seems like this one should be removed.

Reply:
I think we can just have one for the package